src/game/TileEngine/Isometric_Utils.cc

```diff
@@ -137,101 +137,72 @@ void FromScreenToCellCoordinates( INT16 sScreenX, INT16 sScreenY, INT16 *psCellX
 	*psCellY = floor((FLOAT(sScreenY) / HALF_TILE_HEIGHT - FLOAT(sScreenX) / HALF_TILE_WIDTH) / 2.0f);
 }
 
-BOOLEAN GetMouseXY( INT16 *psMouseX, INT16 *psMouseY )
+BOOLEAN GetMouseXY(INT16 *psMouseX, INT16 *psMouseY)
 {
 	INT16 sWorldX, sWorldY;
-
-	if ( !GetMouseWorldCoords( &sWorldX, &sWorldY ) )
-	{
-		(*psMouseX) = 0;
-		(*psMouseY) = 0;
-		return( FALSE );
+	if(!GetMouseWorldCoords(&sWorldX, &sWorldY)) {
```

Comment:
Please don't remove the initial space, it's more readable with it:
`if (!GetMouseWorldCoords(&sWorldX, &sWorldY)) {`